### ISSUE TITLE
Bounding box layer

### ISSUE THREAD

**Issue by groot-1313** [experiencor/basic-yolo-keras#169]
Hi, 

I have a working model using a custom neural network. It is actually an object segmentation model, the last layer of which is a convolutional layer whose output is the half the size of the input image with binary labels for objects. I want to convert my segmentation output to bounding boxes and I think the last layer of YOLO does just that.
But I dont know whether I need all the functionalities of the YOLO network that has been implemented here, so if someone could please hint toward what needs to be done, I would be grateful.
I also feel the last layer of YOLO v1 would work in this case, any other suggestions are welcome.

**Comment by experiencor** [comment]
@groot-1313 My bet would be replacing the last classification label of your model with the detection layer of YOLOv2.
